aqua-organisation/ah-django-Aqua
# #164691225 user can like and dislike an article

#### What is the PR about?
 _to get the following endpoints working to enable user like and dislike articles_
- POST ```/articles/<slug:slug>/dislike/```
- POST ```/articles/<slug:slug>/like/```
#### Description of tasks to be completed?
- Ensure the user can like an article
- Ensure the user can dislike an article
- Ensure user can like their own article
- Ensure article likes and dislikes are counted and returned with the article

#### How should you manually test this?
- one should clone this repository
- one should run ```py manage.py makemigrations``` and also ```py manage.py migrate``` to update table
- run the server ``` py manage.py runserver```

no Body is required only the slug in the endpoint as shown below:

``` 
{
    "article": {
        "article": "Lorem ipsum dolor",
        "username": "nicksbros",
        "likes": true,
        "created_at": "2019-04-21 10:58:08.840832+00:00"
    }
}
```

dislike

``` 
{
    "article": {
        "article": "Lorem ipsum dolor",
        "username": "nicksbros",
        "details": {
            "likes": false,
            "created_at": "2019-04-21 10:58:08.840832+00:00"
        }
    }
``` 


## What are the relevant PT stories?
https://www.pivotaltracker.com/story/show/164691225
